src/envs/behavior.py

```diff
@@ -21,9 +22,14 @@
     from igibson.robots.behavior_robot import BRBody
     from igibson.activity.bddl_backend import SUPPORTED_PREDICATES, \
         ObjectStateUnaryPredicate,ObjectStateBinaryPredicate
+    from igibson.utils.checkpoint_utils import save_checkpoint, load_checkpoint
 
     _BEHAVIOR_IMPORTED = True
     bddl.set_backend("iGibson")  # pylint: disable=no-member
+    if not os.path.exists("tmp_behavior_states/"):
```

Comment:
Use "exist_ok" argument instead of this if (see other usages in this repo for examples)